src/reaktoro_pse/tutorials/softening_acid_ro_example.ipynb

```diff
@@ -0,0 +1,1106 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reaktoro for modeling softening and acid addition for desalination systems\n",
+    "\n",
+    "\n",
+    "#### Dependencies\n",
+    "* Python - Programming language\n",
+    "* Pyomo - Python package for equation-oriented modeling\n",
+    "* IDAES - Python package extending Pyomo for flowsheet modeling\n",
+    "* cyipopt - Solver necessary for use with gray box models\n",
+    "* Reaktoro-pse - Python package for building Reaktoro gray box models\n",
+    "* watertap-solvers - for cyipopt solver wrapper"
+   ]
```

Comment:
I am not sure if we want the dependencies section first. I guess it is useful for others in the future, but maybe not for the audience targeted at the PSE meeting.

So maybe we move this to the very end, and point users to the fact that it is there.

Reply:
I think its fine. I will just skip it during the talk